### PR TITLE
[ALLUXIO-2752] Make CapacityTest final

### PR DESCRIPTION
**Pull request by duwenqin123**
https://alluxio.atlassian.net/browse/ALLUXIO-2752

### CODE REVIEW

**Comment by duwenqin123** [comment]
[ALLUXIO-2752] Make CapacityTest final

**Comment by duwenqin123** [comment]
alluxio-bot, check this please

**Comment by duwenqin123** [comment]
alluxio-bot, check this please

**Comment by duwenqin123** [comment]
alluxio-bot, check this please

**Comment by duwenqin123** [comment]
alluxio-bot, check this please

**Comment by duwenqin123** [comment]
alluxio-bot, check this please

**Comment by duwenqin123** [comment]
alluxio-bot, check this please

File: core/common/src/test/java/alluxio/wire/CapacityTest.java [hunk 1]
```diff
@@ -1,6 +1,4 @@
 /*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
```
Comment (aaudiber) [inline core/common/src/test/java/alluxio/wire/CapacityTest.java]: Please un-delete these two lines

**Comment by yupeng9** [comment]
@duwenqin123 can you sign the CLA and address the comments?

**Comment by aaudiber** [comment]
Closing due to inactivity
@duwenqin123 feel free to re-open if you have time to sign the CLA and address the comments

**Pull request closed without merging**